www/front_src/src/Resources/Details/Body/tabs/Details/DetailsCard/cards.tsx

```diff
@@ -148,7 +154,7 @@ const getDetailCardLines = (
       getLines: (): Lines => [
         {
           key: 'flapping',
-          line: <DetailsLine line={details.flapping ? 'N/A' : labelYes} />,
+          line: <DetailsLine line={details.flapping ? labelYes : 'N/A'} />,
```

Comment:
if flapping is false, it will display "N/A" ?

Reply:
Yes, the rule is -> defined ? show the card ? truish? display 'Yes', 'N/A' otherwise

Reply:
IMHO, it should always display the card with this rule : 
- not defined : N/A
- defined and true : yes
- defined and false : no

(currently, flapping seems not returned by API...)
we may need to check with @Moujimouja what is exactly the behaviour we want

Reply:
@kduret it is actually returned in some cases.

Reply:
there is no any information related to flapping in backend serializer : https://github.com/centreon/centreon/blob/master/config/packages/serializer/Centreon/Monitoring.Resource.yml
This information should be returned in any cases